### DISCUSSION TITLE
{"type":"https://tools.ietf.org/html/rfc7231#section-6.5.1","title":"Bad Request","status":400,"traceId":"00-aaeebc6999cf2956e6de94a1d38cca70-f3458a557f65bd38-00"}

### DISCUSSION THREAD

**Discussion by boli8845** [community/community#36844] [Other Feature Feedback, Questions, & Ideas]
Hi 
I try to run flask run for my pages but keep getting this message.
{"type":"https://tools.ietf.org/html/rfc7231#section-6.5.1","title":"Bad Request","status":400,"traceId":"00-aaeebc6999cf2956e6de94a1d38cca70-f3458a557f65bd38-00"}

Can anyone help me with it?

**Comment by ossanna16** [comment]
Thank you for asking a good question @boli8845, I'm glad you found our community 🙂

**Comment by yoannchaudet** [comment]
Any chances you can share more details? Maybe link to a build where you see this happening?
